src/Catalyst.Core.Lib/Network/DevDnsClient.cs

```diff
@@ -71,7 +71,9 @@ public async Task<IDnsQueryResponse> GetTxtRecordsAsync(string hostname = "seed1
         }
         
         /// <inheritdoc />
+#pragma warning disable 1998
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not suppress issues.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4416347)